quill-core/src/main/scala/io/getquill/util/Show.scala

```diff
@@ -5,6 +5,12 @@ object Show {
     def show(v: T): String
   }
 
+  object Show {
```

Comment:
The code coverage checks are failing because this code is unused. What's the plan for it?